### ISSUE TITLE
Change content of delegate instance

### ISSUE THREAD

**Issue by dpDesignz** [atomiks/tippyjs#767]
I'm creating a table dynamically with javascript, so I've used the [delegate instances as per the documentation](https://atomiks.github.io/tippyjs/v6/addons/), however, I can't work out how to change the content of one of the instances? Can someone please point me in the right direction?

I'm using v6 via the CDN.

**Current delegate code**
```javascript
// Delegate tippy
const delegateInstances = tippy.delegate("#order__lines-table", {
  target: "[data-tippy-content]"
});
```

So one of the changes I'm doing is when you click a button its state toggles and I want the tippy content to update. I had done this like follows, but they just stack on top of each other, the old content doesn't get removed.

```javascript
const viewBtnIcon = viewBtn.querySelector('i');
viewBtnIcon.dataset.tippyContent = `View ${data.name}`;
tippy(viewBtnIcon, {
  content: `View ${data.name}`
});
```

Basically I need to know how to target a specific instance and update the content. How can I do this please?

**Comment by atomiks** [comment]
The instance gets attached as `element._tippy` to the DOM node, so you can querySelect it and use that property.

**Comment by dpDesignz** [comment]
> The instance gets attached as `element._tippy` to the DOM node, so you can querySelect it and use that property.

Are you able to post some examples of code please? Because there could be 30 or 40+ instances in here, all assigned by the delegate, how do I reference 1? I can post more code examples from my end if required, I just need to know what you need?

**Comment by atomiks** [comment]
```js
const viewBtnIcon = viewBtn.querySelector('i');
viewBtnIcon._tippy.setContent(`View ${data.name}`);
```

**Comment by dpDesignz** [comment]
Ok, I now understand how to reference it, how do I now change the content? I've tried the following code, but I still have the issue of the old content not being replaced and I now end up with 2 Tippy instances attached to the 1 element?

```javascript
const changeBtn = document.querySelector("#line_1_change");
tippy(changeBtn, {
  allowHTML: true,
  delay: 200,
});
const instance = changeBtn._tippy;
instance.setContent("Updated content!");
```

I have a working [jsfiddle example here](https://jsfiddle.net/dpdesignz/cnr3s4md/)

Also, am I correct that I have to assign the props again? It won't just pull it from the existing props?

**Comment by dpDesignz** [comment]
> 
> 
> ```js
> const viewBtnIcon = viewBtn.querySelector('i');
> viewBtnIcon._tippy.setContent(`View ${data.name}`);
> ```

Thank you so much! This works perfectly as expected 😄
